src/test/scala/org/camunda/feel/impl/SuppressedFailuresTest.scala

```diff
@@ -49,48 +49,48 @@ class SuppressedFailuresTest extends AnyFlatSpec
   it should "report a suppressed failure if input is not comparable with interval" in {
     evaluateUnaryTests("[2..5]", "NaN") should reportFailure(
       failureType = EvaluationFailureType.NOT_COMPARABLE,
-      failureMessage = "Can't compare ValString(NaN) with ValNumber(2) and ValNumber(5)"
+      failureMessage = """Can't compare '"NaN"' with '2' and '5'"""
     )
   }
 
   it should "report a suppressed failure if values are not comparable" in {
     evaluateExpression("true < 2") should reportFailure(
       failureType = EvaluationFailureType.NOT_COMPARABLE,
-      failureMessage = "Can't compare ValBoolean(true) with ValNumber(2)"
+      failureMessage = "Can't compare 'true' with '2'"
     )
   }
 
   it should "report a suppressed failure if an addition has incompatible values" in {
     evaluateExpression("2 + true") should reportFailure(
       failureType = EvaluationFailureType.INVALID_TYPE,
-      failureMessage = "Expected Number but found 'ValBoolean(true)'"
+      failureMessage = "Expected number but found 'true'"
```

Comment:
❓ How this is possible ? 🪄 

```Number -> number```

Reply:
I aligned the failure messages and changed the type names to start with a lowercase. See [here](https://github.com/camunda/feel-scala/blob/269a60fab0c227988c589a5ef217b114bc8a8243/src/main/scala/org/camunda/feel/impl/interpreter/FeelInterpreter.scala#L304). :rocket: